### ISSUE TITLE
Unloading Stylesheets loaded from .css files

### ISSUE THREAD

**Issue by TheJhonny007** [edvin/tornadofx#834]
I have designed multiple stylesheets in css files and want to swap them out according to user preferences.
I can't find a method to unload a specific stylesheet which was loaded from the file system.

Is there a method to swap out stylesheets which are defined in .css files on demand?

**Comment by edvin** [comment]
Yes, you can add/remove stylesheets with the `importStylesheet()` and `removeStylesheet()` functions, or you can manually manipulate the list of URL's in the `Scene.stylesheets` property.
